Sources/Location/Extensions/Location.CLCircularRegion.swift

```diff
@@ -9,7 +9,7 @@
 import CoreLocation
 
 extension CLCircularRegion: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.